pkg/corerp/frontend/controller/applications/getgraph_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package applications
+
+import (
+	"context"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	v1 "github.com/radius-project/radius/pkg/armrpc/api/v1"
+	ctrl "github.com/radius-project/radius/pkg/armrpc/frontend/controller"
+	"github.com/radius-project/radius/pkg/armrpc/rpctest"
+	"github.com/radius-project/radius/pkg/sdk"
+	"github.com/radius-project/radius/pkg/ucp/store"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetApplicationGraphRun_20231001Preview(t *testing.T) {
+	mctrl := gomock.NewController(t)
+	defer mctrl.Finish()
+
+	mStorageClient := store.NewMockStorageClient(mctrl)
+	req, err := rpctest.NewHTTPRequestWithContent(
+		context.Background(),
+		v1.OperationPost.HTTPMethod(),
+		"http://localhost:8080/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/radius-test-rg/providers/Applications.Core/Applications/myapp/getGraph?api-version=2023-10-01-preview", nil)
+
+	require.NoError(t, err)
+
+	t.Run("resource not found", func(t *testing.T) {
```

Comment:
Are we going to have more tests here?

Reply:
not sure how my older comment got lost. Its been a bit tricky to add other tests, since the mocking happens on a lower level. I have added UT for 404 and E2E captures the success. We can discuss if you think I can do something else.